### ISSUE TITLE
[Feature Request]: env for component prefix

### ISSUE THREAD

**Issue by chrillep** [tasteui/tasteui#351]
### Suggestion Theme

when updating to 1.20 the config broke since it has a different list of components in the config.
The only reason we publoshed the config was because of the prefix.
If the prefix would use an ENV TALLSTACKUI_PREFIX by default, we wouldnt have to publish the config at all since we use all the defaults.

Consider it ❤️ 

https://github.com/tallstackui/tallstackui/blob/1.x/src/config.php#L17

```php
'prefix' => env('TALLSTACKUI_PREFIX'),
```

### Main Benefits

the goal would/should be not to have to publish the config.

### Source of the Suggestion

https://github.com/tallstackui/tallstackui/blob/1.x/src/config.php#L17

**Comment by devajmeireles** [comment]
Hey, @chrillep ! Thanks for your feature request.

I'm not really in favor of doing this by default, even from v1.20 we have the command:

```bash
php artisan tallstackui:setup-prefix
```

However, if you want to not use the command and use an environment variable, just do what you did.

1. Create the variable in `.env`
2. Use the variable in the configuration file: `'prefix' => env('TALLSTACKUI_PREFIX'),`

Maybe in future v2, I'll make this by default 👍🏻

**Comment by chrillep** [comment]
thats fine, i just wanted to share my dealings :) 
so just to be clear, when breaking changes in config

run

```sh
rm config/tallstackui.php
php artisan tallstackui:setup-prefix
```

**Comment by devajmeireles** [comment]
Hey, @chrillep !

The `setup-prefix` command will publish `config/tallstackui.php` and set the prefix. You can publish the configuration file and instead of using the setup prefix command, follow the tutorial above to set the prefix via the env variable.

I will reconsider your feature request to configure a prefix without needing to publish the configuration file and set up the prefix by using an env variable 👍🏻

**Comment by devajmeireles** [comment]
Hey, @chrillep ! We just released v1.20.1 with this feature. I'll update the docs soon.

You can now use a env variable to set prefix:

```env
TALLSTACKUI_PREFIX="ts-"
```

**Comment by chrillep** [comment]
Thank you! We are really enjoying the work you do! Keep up the good work ! ❤️
